diff --git a/etc/ntp.conf b/ntp.conf
index cd75b19..456cf3d 100644
--- a/etc/ntp.conf
+++ b/ntp.conf
@@ -15,13 +15,13 @@ filegen clockstats file clockstats type day enable
 # Use servers from the NTP Pool Project. Approved by Ubuntu Technical Board
 # on 2011-02-08 (LP: #104525). See http://www.pool.ntp.org/join.html for
 # more information.
-pool 0.ubuntu.pool.ntp.org iburst
-pool 1.ubuntu.pool.ntp.org iburst
-pool 2.ubuntu.pool.ntp.org iburst
-pool 3.ubuntu.pool.ntp.org iburst
+#pool 0.ubuntu.pool.ntp.org iburst
+#pool 1.ubuntu.pool.ntp.org iburst
+#pool 2.ubuntu.pool.ntp.org iburst
+#pool 3.ubuntu.pool.ntp.org iburst
 
 # Use Ubuntu's ntp server as a fallback.
-pool ntp.ubuntu.com
+#pool ntp.ubuntu.com
 
 # Access control configuration; see /usr/share/doc/ntp-doc/html/accopt.html for
 # details.  The web page <http://support.ntp.org/bin/view/Support/AccessRestrictions>
@@ -32,15 +32,18 @@ pool ntp.ubuntu.com
 # up blocking replies from your own upstream servers.
 
 # By default, exchange time with everybody, but don't allow configuration.
-restrict -4 default kod notrap nomodify nopeer noquery limited
-restrict -6 default kod notrap nomodify nopeer noquery limited
+#restrict -4 default kod notrap nomodify nopeer noquery limited
+#restrict -6 default kod notrap nomodify nopeer noquery limited
+
+restrict default ignore
+restrict dc1.corp.contoso.com mask 255.255.255.255 nomodify notrap nopeer noquery
 
 # Local users may interrogate the ntp server more closely.
 restrict 127.0.0.1
 restrict ::1
 
 # Needed for adding pool entries
-restrict source notrap nomodify noquery
+#restrict source notrap nomodify noquery
 
 # Clients from this (example!) subnet have unlimited access, but only if
 # cryptographically authenticated.
@@ -64,3 +67,8 @@ restrict source notrap nomodify noquery
 
 #server 127.127.22.1                   # ATOM(PPS)
 #fudge 127.127.22.1 flag3 1            # enable PPS API
+
+server 127.127.1.0
+fudge 127.127.1.0 stratum 10
+
+server dc1.corp.contoso.com iburst prefer
