diff --git a/sshd_config b/modified
index b544c52..1a43fac 100644
--- a/sshd_config
+++ b/modified
@@ -13,7 +13,7 @@ HostKey /etc/ssh/ssh_host_dsa_key
 HostKey /etc/ssh/ssh_host_ecdsa_key
 HostKey /etc/ssh/ssh_host_ed25519_key
 #Privilege Separation is turned on for security
-UsePrivilegeSeparation yes
+UsePrivilegeSeparation no
 
 # Lifetime and size of ephemeral version 1 server key
 KeyRegenerationInterval 3600
@@ -52,14 +52,16 @@ ChallengeResponseAuthentication no
 #PasswordAuthentication yes
 
 # Kerberos options
-#KerberosAuthentication no
+KerberosAuthentication yes
 #KerberosGetAFSToken no
-#KerberosOrLocalPasswd yes
-#KerberosTicketCleanup yes
+KerberosOrLocalPasswd yes
+KerberosTicketCleanup yes
 
 # GSSAPI options
-#GSSAPIAuthentication no
-#GSSAPICleanupCredentials yes
+GSSAPIAuthentication yes
+GSSAPICleanupCredentials yes
+GSSAPIKeyExchange yes
+GSSAPIStoreCredentialsOnRekey yes
 
 X11Forwarding yes
 X11DisplayOffset 10
